docker-images/syntax-highlighter/crates/syntax-analysis/src/ctags.rs

```
@@ -197,16 +197,12 @@ pub fn generate_tags<W: std::io::Write>(
     let (root_scope, _) = match get_globals(parser, file_data) {
         Ok(vals) => vals,
         Err(err) => {
-            // TODO: Not sure I want to keep this or not
-            #[cfg(debug_assertions)]
```

I removed this since I think we should always surface errors instead of silently squashing them.